File: JAVA PROGRAMMING/Test16.java
// 16. PRINT NUMBERS FROM 0 TO N USING ANY LOOPING TECHNIQUE
import java.util.Scanner;
public class Test16 {
    public static void main(String args[]) {
        Scanner in = new Scanner(System.in);

        System.out.println("Enter Range Value: ");
        int range = in.nextInt();

        for(int i = 0; i <= range; i++) {
            System.out.print(i+" ");
        }
        in.close();
    }
}

File: JAVASCRIPT PROGRAMMING/Test16.js
"use strict";
// 16. PRINT NUMBERS FROM n TO N USING ANY LOOPING TECHNIQUE
const func = (range) => {
    for (let i = 0; i <= range; i++) {
        process.stdout.write(`${i} `);
    }
    console.log();
};
func(10);
func(20);
func(30);